webui/react/src/pages/ExperimentDetails/ExperimentDetailsHeader.tsx

```diff
@@ -235,6 +238,12 @@ const ExperimentDetailsHeader: React.FC<Props> = ({
                 {shortEnglishHumannizer(getDuration(experiment))}
               </div>
             )}
+            {experiment.jobSummary !== undefined && (
```

Comment:
**suggestion:** I think `jobSummary` can only be `undefined` or an object so it should be ok to just have:
```
{experiment.jobSummary && (
```